nautobot/dcim/templates/dcim/controllerdevicegroup_create.html

```diff
@@ -0,0 +1,43 @@
+{% extends 'generic/object_create.html' %}
+{% load form_helpers %}
+{% load helpers %}
+{% load static %}
+
+{% block javascript %}
+<script type="text/javascript">
+    const parentInput = $('select#id_parent');
```

Comment:
Should I refactor this not to be JQuery dependent? @glennmatthews

Reply:
If you *can* - the `select2` library that we use for most select widgets only emits jQuery events so we might have to keep at least some of the jQuery logic here.

Reply:
Keeping as is